pyproject.toml

```diff
@@ -110,3 +112,6 @@ omit = ["*/tests/*"]
 
 [tool.pytest.ini_options]
 asyncio_mode = "auto"
+
+[tool.poetry.scripts]
+starlite-cli = "starlite.cli:cli"
```

Comment:
Can you run pre-commit over all this? Instructions in `CONTRIBUTING.md`.

Reply:
Yes sure, forgot that sorry - will be done when I develop further